packages/@aws-cdk/aws-lambda-event-sources/lib/stream.ts

```diff
@@ -21,6 +21,47 @@ export interface StreamEventSourceProps {
    */
   readonly batchSize?: number;
 
+  /**
+   * If the function returns an error, split the batch in two and retry.
+   *
+   * @default false
+   */
+  readonly bisectBatchOnFunctionError?: boolean;
+
+  /**
+   * An Amazon SQS queue or Amazon SNS topic destination for discarded records.
+   *
+   * @default discarded records are ignored
+   */
+  readonly destinationConfig?: lambda.CfnEventSourceMapping.DestinationConfigProperty | IResolvable | undefined;
+
+  /**
+   * The maximum age of a record that Lambda sends to a function for processing.
+   * Valid Range:
+   * * Minimum value of 60
+   * * Maximum value of 604800
+   *
+   * @default 604800
```

Comment:
The default doesn't seem to be set.

Reply:
I wasn't sure how the defaults were being documented.  If the parameter is not provided, right now the default of the created component is 604800.  So instead of having it get set in the CFT that was created I let the platform handle it.  Not sure if this is the pattern that's been followed in the past or not.

Reply:
We should be documenting the defaults here even if we're letting CloudFormation pick the default.

Reply:
They are documented, they're just not being set.  I completely agree from the documentation perspective.  I'm just not making sure it was set to the default when it's passed through the library since CloudFormation is picking it.  Does that make sense?